docs/contributing/documenting-instrumentation.md

```diff
@@ -190,8 +190,22 @@ within 24 hours.
 
 ## opentelemetry.io
 
-All of our instrumentation modules are listed on the opentelemetry.io website in the context of how
-to [suppress specific instrumentation](https://opentelemetry.io/docs/zero-code/java/agent/disable/#suppressing-specific-agent-instrumentation).
+All of our instrumentation modules are listed on the opentelemetry.io website in two places:
+
+### Supported Libraries
+
+The [Supported Libraries](https://opentelemetry.io/docs/zero-code/java/agent/supported-libraries/)
+page lists all the library instrumentations that are included in the OpenTelemetry Java agent. It
+mostly mirrors the information from the [supported libraries](../supported-libraries.md) page in
```

Comment:
are you thinking to remove this page eventually? (leaving behind a stub like https://github.com/open-telemetry/community/blob/main/community-membership.md since there are probably a lot of references to it)

Reply:
I wasn't thinking of replacing it, but I had thought of automating the generation/management of it. Personally, I like having it within the repo.